moped-database/migrations/1644104957483_drop_moped_proj_features_moped_project_relations/up.sql

```diff
@@ -0,0 +1,4 @@
+
+alter table "public"."moped_proj_features" drop constraint "moped_proj_features_project_id_fkey";
+
+ALTER TABLE "public"."moped_proj_features" DROP COLUMN "project_id" CASCADE;
```

Comment:
i have doubts about the need for CASCADE here. and in fact i'm not sure why it would be necessary. no other table or view in the db should depend this column 🤔

Reply:
It is totally possible that I have drug a column into the refactor by accident. Adding too few columns would not have made it out the gate, but adding too many is hard to detect in testing.

They were chosen purely from trying to delete the first record out the project table and reworking constraints adding cascades until it was possible. This process was then repeated for the second, adding any cascades until that was possible, and then repeating per record until there were no more records in the projects table. 

That's not to say that I didn't think about what a change was going to do when I put it in a commit, but truely, there is *plenty of room* for error between my relatively new knowledge of the actual meaning behind each table and how they relate to each other.

Reply:
no worries—`project_id` has been in use on this table for as long as components have existed 😄 

i'm just not sure why i stuck `cascade` on this `DROP` statement. i'm quite sure i added it after misunderstanding a Hasura Console error due to the hasura metadata relationships blocking the column drop. but because i don't understand the need to cascade i think i should remove it and test the migration again.

Reply:
Goodness, please forgive my comment! I saw CASCADE and an observation on a change to the schema, and my mind jumped back to the changes in https://github.com/cityofaustin/atd-moped/pull/539. Totally unrelated - 🐦 🧠 .